components/image/lazyLoad/src/index.js

```diff
@@ -10,17 +10,25 @@ const BASE_CLASS = 'sui-ImageLazyLoad'
  */
 export default function ImageLazyLoad({
   alt = '',
-  title = '',
   aspectRatio = '',
-  onError = () => {},
+  isContained = false,
   offsetVertical = 100,
+  onError = () => {},
   showSpinner = true,
-  src
+  src,
+  title = ''
 }) {
   const [isNearScreen, fromRef] = useNearScreen({offset: `${offsetVertical}px`})
 
   const lazyLoadWrapClassName = cx(BASE_CLASS, {
-    [`${BASE_CLASS}--ratio-${aspectRatio.replace(':', '-')}`]: aspectRatio
+    [`${BASE_CLASS}--ratio-${aspectRatio.replace(':', '-')}`]: aspectRatio,
+    [`${BASE_CLASS}--is-contained`]: isContained
+  })
+  const lazyLoadImageWrapClassName = cx(`${BASE_CLASS}-imageWrap`, {
+    [`${BASE_CLASS}-imageWrap--is-contained`]: isContained
+  })
+  const lazyLoadImageClassName = cx(`${BASE_CLASS}-image`, {
+    [`${BASE_CLASS}-image--is-contained`]: isContained
```

Comment:
Following the [SUITCSS Naming](https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md#is-stateOfComponent) convention
```suggestion
    'is-contained': isContained
```